fieldtracing/fieldtracing.cpp

```diff
@@ -1182,34 +1183,28 @@ namespace FieldTracing {
             {
                cellsToDoFullBox = 0;
                cellsToDoFluxRopes = 0;
-               cellFWTracingStepSize.swap(reducedCellFWTracingStepSize);
-               cellBWTracingStepSize.swap(reducedCellBWTracingStepSize);
-               cellFWRunningDistance.swap(reducedCellFWRunningDistance);
-               cellBWRunningDistance.swap(reducedCellBWRunningDistance);
-               cellFWConnection.swap(reducedCellFWConnection);
-               cellBWConnection.swap(reducedCellBWConnection);
-               // No swap for these as the old ones are used for the small arrays ~70 lines up.
-               cellNeedsContinuedFWTracing = reducedCellNeedsContinuedFWTracing;
-               cellNeedsContinuedBWTracing = reducedCellNeedsContinuedBWTracing;
+               // These are used for the small arrays ~70 lines up.
+               reducedCellNeedsContinuedFWTracing = cellNeedsContinuedFWTracing;
```

Comment:
if reductions are only done to the "small" vectors, then should these in fact be renamed to `previousRoundCellNeedsContinuedXWTracing`? or `stored...`?